### PR TITLE
New version: Automattic.Simplenote version 2.19.0

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28201)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
Hmm, not sure whats different or where it went wrong

File: manifests/a/Automattic/Simplenote/2.19.0/Automattic.Simplenote.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Platform:
+  - Windows.Universal
+  Architecture: x64
+  InstallerType: appx
+  Scope: user
+  InstallerUrl: https://github.com/Automattic/simplenote-electron/releases/download/v2.19.0/Simplenote-win-2.19.0-x64.appx
+  InstallerSha256: C65BEB5BEF167977A83834BEABBB2A82002645BDB6DDD4B4C79F846F74A8832B
+  SignatureSha256: 775B03E5C7F275F09F4DA2A902040F4674775CD07802E233144EF7A70D20C56A
```
Comment (ItzLevvie) [inline manifests/a/Automattic/Simplenote/2.19.0/Automattic.Simplenote.installer.yaml:37]:
```suggestion
  SignatureSha256: 634979567715DC67D4B4EED16AE52DEC094D715140A88DCBA5A6268C3606B8BA
```

File: manifests/a/Automattic/Simplenote/2.19.0/Automattic.Simplenote.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  PackageFamilyName: simplenote_cdagv53c87npa
+- Platform:
+  - Windows.Universal
+  Architecture: x86
+  InstallerType: appx
+  Scope: user
+  InstallerUrl: https://github.com/Automattic/simplenote-electron/releases/download/v2.19.0/Simplenote-win-2.19.0-ia32.appx
+  InstallerSha256: 317DA9AFB16051FBEB8AC8A73C8B739BEC8F9786E5BFFA9BFB03FB4DFC47E6DB
+  SignatureSha256: BE7B703FD898EBCE7DA5E239121E0F9DC480F3FB1D39AF9775CA27381CF4A84F
```
Comment (ItzLevvie) [inline manifests/a/Automattic/Simplenote/2.19.0/Automattic.Simplenote.installer.yaml:46]:
```suggestion
  SignatureSha256: 1A157EE9819A55EE064CCF7443E6A8271CB44CEF8E711E6DA994223BC1B1A2C1
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/Automattic/Simplenote/2.19.0/Automattic.Simplenote.installer.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/a/Automattic/Simplenote/2.19.0/Automattic.Simplenote.installer.yaml:37]: wut

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**